subprojects/launcher/src/main/java/org/gradle/launcher/daemon/client/DaemonClient.java

```diff
@@ -139,9 +139,13 @@ public BuildActionResult execute(BuildAction action, BuildActionParameters param
         LOGGER.debug("Executing build {} in daemon client {pid={}}", buildId, processEnvironment.maybeGetPid());
 
         int saneNumberOfAttempts = 100; //is it sane enough?
```

Comment:
I think we could probably reduce the number of times we retry now, but let's leave this here for now.

Reply:
👍